### DISCUSSION TITLE
Minimap

### DISCUSSION THREAD

**Discussion by christian-byrne** [Comfy-Org/ComfyUI_frontend#181] [Ideas]
![NodeMinimap-low](https://github.com/user-attachments/assets/0e0f533e-6aa4-4dd1-a6ce-21eab6192e00)

Idea from MoonL88537 on Twitter.

Seems very hard to implement.

**Comment by huchenlei** [comment]
I know Reactflow/Vueflow has the mini map built-in. If later we decide to migrate to it, we can get this feature for free, but if we stick to litegraph, this is some serious engineering work get the feature.

**Reply by christian-byrne** [reply]
Wow, the way the Vlueflow minimap works is really cool

**Comment by toyxyz** [comment]
https://github.com/OliverCrosby/Comfyui-Minimap
